src/CitMovie.Api/Program.cs

```diff
@@ -50,11 +50,11 @@
 builder.Services.AddControllers();
 builder.Services.AddScoped<IFollowRepository, FollowRepository>();
 builder.Services.AddScoped<FollowService>();
+builder.Services.AddScoped<ILanguageRepository, LanguageRepository>();
+builder.Services.AddScoped<LanguageManager>();
```

Comment:
Please use the `CitMovieServiceCollectionExtensionsAddCitMovieServices()` method in the business layer to register dependencies

Reply:
Nice I have not realized we have this :D was thinking we should do something like that